web/app_options.js

```diff
@@ -187,9 +187,14 @@ const defaultOptions = {
   },
   enableAltText: {
     /** @type {boolean} */
-    value: false,
+    value: true,
```

Comment:
Tracking comment to make sure the changes in this file get updated before merging (it's also what makes linting fail now).